com.synology.CloudStationBackup.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.synology.CloudStationBackup
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: synology-cloud-station-backup
+tags: [proprietary]
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
```

Comment:
Does it actually work natively on Wayland or drops to xwayland?

Reply:
Looks like it's dropping to xwayland. I'll remove and test a build without the wayland socket. I put it in because the existing synology flatpak had it as well.

Reply:
note: you don't need to remove and create a new build to test. You can revoke the permission in your flatpak run command with --nosocket=wayland

Reply:
That's a bit easier. Thanks for the information.

Reply:
removed 
`- --socket=wayland`